Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+
+# Comment verrouiller votre SportShield
+
+SportShield protège vos affaires de sport contre les voleurs. Son installation n'a que quelques étapes:
+
+1. Tirez le câble de la sortie de câble du SportShield.
+2. Entourez vos affaires avec le câble.
+3. Attachez le câble au porte-câble.<br>
+<img src="./Img/cableHolderAndExitFrench.png" width="200"/>
+
+Si SportShield n'est pas connecté à votre smartphone ou avec une carte NFC, le dispositif ne fonctionnera pas comme prévu.<br>
+Réferez-vous à [partie initialisation](#initialisation-de-votre-sportshield).<br><br>
+SportShield est maintenant verrouillé et vous ne pourrez plus tirer sur le câble si vous ne le déverrouillez pas. <br><br>
+Vous recevrez une notification si le SportShield détecte un choc. <br>
+Pour plus d'informations sur les notifications, reportez-vous à la [section dédiée](#notifications).
+
+<div style="page-break-after: always;"></div>
+
+# Comment déverrouiller votre SportShield
+
+Le déverrouillage de SportShield est plutôt simple. Vous pouvez le déverrouiller par Bluetooth ou à l'aide de l'application. <br>
+Par Bluetooth:
+
+1. Allez dans vos paramètres Bluetooth.<br>
+<img src="./Img/BluetoothSettingsFunctionnalities.png" width="200"/>
+2. Cliquez sur le bouton "Unlock" pour déverrouiller le SportShield.<br>
+<img src="./Img/BluetoothUnlock.png" width="200"/>
+
+Avec une carte NFC:
+
+- Rentrez la carte NFC en contact avec votre SportShield.
+
+A propos de l'app SportShield, reportez-vous à la [section dédiée](#app).
+
+<br>
+SportShield est maintenant déverrouillé. L'alarme ne sonnera pas et vous pouvez reprendre le dispositif.
+
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# Alarme
+
+Quand votre SportShield est verrouillé, chaque choc détecté sonnera une alarme. <br>L'intensité de l'alarme dépend de l'intensité du choc: <br>
+
+- 3 sons légers lorsqu'un choc léger est détecté.
+  
+- 5 sons longs et aigus lorsqu'un choc important est détecté.
+<br><br>
+
+De plus, vous pouvez arrêter l'alarme quand vous le souhaitez. Pour ce faire, vous pouvez utiliser le Bluetooth ou l'application. <br>
+
+Par Bluetooth: ce sont les mêmes étapes que le [verrouillage du SportShield](#comment-verrouiller-votre-sportshield).
+
+Avec une carte NFC, ce sont les mêmes étapes que le [déverrouillage du SportShield](#comment-déverrouiller-votre-sportshield).
+
+A propos de l'application SportShield, reportez-vous à la [partie dédiée](#app).
+
+Quand l'alarme sonne, une notification est envoyé au smartphone pairé.<br>
+Pour plus d'informations sur les notifications, reportez-vous à la [section dédiée](#notifications).
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# App
+
+L'application SportShield serait disponible sur l'Apple App Store et le Google Play Store.<br>
+Même si l'application n'a pas encore été synchronisée avec l'appareil SportShield. <br><br>Dans ce manuel, vous ne trouverez pas pour l'instant d'informations utiles sur cette application.
```

Comment:
remove "même si" and change "vous ne trouverez pas pour l'instant d'informations utiles" by "pour le moment, aucune informations utiles"